README.md

```diff
@@ -1,19 +1,74 @@
-# Drools 
+## Prerequisites
 
-## Installing and Running Kakfa Cluster
-- Kafka Cluster on Openshift on fedora
-https://github.com/desmax74/openshift-handbook/blob/master/fedora/kafka.md
-- Kafka Cluster on Openshift on ubuntu
-https://github.com/desmax74/openshift-handbook/blob/master/ubuntu/kafka.md
+- Openshift 3.11 or Minishift
 
+- A Kafka Cluster on Openshift 3.11 with Strimzi https://strimzi.io/
+(tested on Openshift 3.11 and strimzi 0.11.1)
+
+## Creation of Kafka's topics
+Create the kafka topics using the files in the kafka-topics folder, 
+the cluster's name y default is "my-cluster", change accordingly in 
+the yaml files with your cluster's name 
+```sh
+oc create -f events.yaml
+oc create -f control.yaml
+oc create -f snapshot.yaml
+oc create -f kiesessioninfos.yaml
+```
+Checks the topics
+```sh
+oc exec -it my-cluster-kafka-0 -- bin/kafka-topics.sh --zookeeper localhost:2181 --describe
+```
+
+#### Pre deploy on Openshift
+Relax RBAC for configmap
 ```sh
-mvn clean package
+kubectl create clusterrolebinding permissive-binding --clusterrole=cluster-admin --group=system:serviceaccounts
+```
+
+## Build the pods
+```sh
+mvn clean install -DskipTests
 ```
 ### Deployment
+Are available three modules 
 
 - Springboot     (openshift-kie-springboot.jar 52,5 mb)
 - Thorntail      (openshift-kie-thorntail.jar 111 mb)
 - Jdk HttpServer (openshift-kie-jdkhttp.jar 36,9 mb)
```

Comment:
Please remove jar size here because I'm sure it will be not maintained so it will be wrong quickly (maybe it is already wrong)

Reply:
Ok